userguide/system.rst

```diff
@@ -2151,14 +2155,14 @@ notifications as the reports are actioned.
 Before creating a bug report or feature request, ensure that an
 existing report does not already exist at
 https://redmine.ixsystems.com/projects/freenas/issues.
-If you find a similar issue that is not yet marked as *closed* or
-*resolved*, add a comment to that issue if you have new information
-to provide that can assist in resolving the issue. If you find a
-similar issue that is marked as *closed* or *resolved*, you can
+If there is a similar issue that is not yet marked as *closed* or
+*resolved*, add a comment to that issue if there is new information
+to provide that can assist in resolving the issue. If there is a
+similar issue that is marked as *closed* or *resolved*,
 create a new issue and refer to the earlier issue number.
 
-.. note:: If you are not updated to the latest version of STABLE,
-   do that first to see if it resolves your issue.
+.. note:: Update the system to the latest version of STABLE
+   first to see if it resolves the issue.
 
```

Comment:
Change the note to:
Always update the system to the latest version of STABLE to see if the issue is resolved prior to reporting a bug.